feat(clickhouse): ClickHouse destination (#6982)
cloudquery/cloudquery: plugins/destination/clickhouse/README.md

> # CloudQuery Microsoft SQL Server Destination Plugin

Reviewer: ```suggestion
# CloudQuery ClickHouse Destination Plugin
```

> This destination plugin lets you sync data from any CloudQuery source to a Microsoft SQL Server database.

Reviewer: ```suggestion
This destination plugin lets you sync data from any CloudQuery source to a ClickHouse database.
```

## Links

> - [User Guide](https://cloudquery.io/docs/plugins/destinations/mssql/overview)

Reviewer: ```suggestion
- [User Guide](https://cloudquery.io/docs/plugins/destinations/clickhouse/overview)
```
